### PR TITLE
OSDOCS-11694 Adding gpu support to Nutanix

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.17

Issue:
https://issues.redhat.com/browse/OSDOCS-11694

Link to docs preview:
https://82093--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installation-config-parameters-nutanix.html

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -3158,6 +3158,30 @@ d|List.
 
 The name of one or more failures domains.
 
+|compute:
+  platform:
+    nutanix:
+      gpus:
+        type:
+|The type of identifier used to attach a GPU to a compute machine. Valid values are "Name" or "deviceID".
```
Comment (yanhua121) [inline modules/installation-configuration-parameters.adoc:3166]: "deviceID" should be "DeviceID".

File: modules/installation-configuration-parameters.adoc [hunk 2]
```diff
@@ -3158,6 +3158,30 @@ d|List.
...
+|String
+
+|compute:
+  platform:
+    nutanix:
+      gpus:
+        name:
+|The name of the GPU device to attach to a compute machine.
```
Comment (yanhua121) [inline modules/installation-configuration-parameters.adoc:3174]: name is required if the type is "Name".

File: modules/installation-configuration-parameters.adoc [hunk 3]
```diff
@@ -3158,6 +3158,30 @@ d|List.
...
+|String
+
+|compute:
+  platform:
+    nutanix:
+      gpus:
+        deviceID:
+|The device identifier of the GPU device to attach to a compute machine. This information is available in Prism Central.
```
Comment (yanhua121) [inline modules/installation-configuration-parameters.adoc:3182]: deviceID is required if the type is "DeviceID".

**Comment by bscott-rh** [comment]
@sgaoshang  PTAL at this PR for GPUs on nutanix. ty

**Review by xenolinux — COMMENTED** [review]
A question related to back ticks; otherwise LGTM

A gentle reminder to add preview link

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

File: modules/installation-configuration-parameters.adoc [hunk 4]
```diff
@@ -3172,6 +3172,30 @@ d|List.
 
 The name of one or more failures domains.
 
+|compute:
+  platform:
+    nutanix:
+      gpus:
+        type:
+|The type of identifier used to attach a GPU to a compute machine. Valid values are "Name" or "DeviceID".
```
[the remark this answers is not in the document]
Comment (bscott-rh) [reply modules/installation-configuration-parameters.adoc:3180]: These should stay as regular quotes, because they're strings, not technical parameters.

**Pull request merged**